CMakeLists.txt

```diff
@@ -184,6 +184,77 @@ foreach (build_config Debug Release RelWithDebInfo MinSizeRel)
   endforeach()
 endforeach()
 
+# Make a target that aggregates a number of other targets;
+# this can be used to group builds (e.g. "build_tests")
+# or to execute targets (by adding executable custom-commands);
+# the latter is used to execute test targets.
+#
+# Note that the target will be excluded from the "all" target, so it won't
+# be built by "make all" by default.
+#
+# TODO: this is intended to eventually replicate all of the interesting test targets
+# from our Make build, but not all are implemented yet:
+# TODO(srj): add test_aotcpp_generators support
+# TODO(srj): add test_valgrind variant
+# TODO(srj): add test_avx512 variant
+# TODO(srj): add test_bazel variant
+# TODO(srj): add test_python variant
+# TODO(srj): add test_apps variant
+# TODO(srj): add run_tests variant
```

Comment:
Is run_tests still a TODO? I see some indication it exists below...

Reply:
Fixed